css-layout-classes/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Layout</title>
+  </head>
+
+  <body>
+
+    <div id="body">
+      <div id="title">
+        <h1>Choose your character!</h1>
+      </div>
+
+
+      <div class="row">
+
+        <div class="container" id="container-one">
+
+          <div id="image">
+            <img src="images/link.png" alt="link from zelda">
+          </div>
+
+          <div class="description">
+            <h3>Super Smash Bros. Ultimate</h3>
+            <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the
+              best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash
+              gameplay elements, although its online mode was widely criticized.</p>
+          </div>
+
+          <div class="character"">
+            <img src="images/yoshi.png" alt="Yoshi">
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="row">
+
+        <div class="container" id="container-two">
+
+          <div class="fighter">
+            <img src="images/kirby.png" alt="Kirby">
+          </div>
+          <div class="fighter">
+            <img src="images/fox.png" alt="Fox">
+          </div>
+          <div class="fighter">
+            <img src="images/pikachu.png" alt="Pikachu">
+          </div>
+          <div class="fighter">
+            <img src="images/ness.png" alt="Ness">
+          </div>
+
+        </div>
+
+      </div>
+
+      <div class="row">
+
+        <div class="container" id="container-three">
+
+          <div class="description">
+            <h3>Donkey Kong</h3>
+            <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by Shigeru
+            Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional spin-off titles of
+            various genres.</p>
+
+            <p>The games of the first genre are mostly single-screen platform/action puzzle types, featuring Donkey Kong as the
+            opponent in an industrial construction setting.
+            </p>
+          </div>
+
+          <div id="donkey-image">
+            <img src="images/dk.png" alt="Donkey Kong">
+          </div>
+        </div>
+
+      </div>
+
+      <div id="fourth-row">
+
+        <div class="container" id="container-four">
+          <div id="mario">
+            <img src="images/mario.png" alt="Brad Pitt">
+          </div>
+
+          <div class="description">
```

Comment:
This should be a class related to a column. So an appropriate name would be `column-two-thirds`